packages/vue/src/components/SpCrudRead/SpCrudRead.vue

```diff
@@ -92,20 +106,32 @@ export default defineComponent({
 
     // computed
     let address = computed(() => $s.getters['common/wallet/address'])
-    let posts = computed(
-        () => $s.state[props.storeName]
+    let itemFields = computed(() => $s.getters[props.storeName + '/getTypeStructure'](props.itemName))
+    let items = computed(
+      () => {
+        const itemData = $s.state[props.storeName][props.itemName + 's']
```

Comment:
To get all items, instead of peeking into vuex state, use a vuex getter like in [here](https://github.com/tendermint/vue/blob/8592c94e43592b17c7367606ace161d80a9fff74/packages/vue/src/components/SpTypeList/SpTypeList.vue#L122)

Reply:
That doesn't seem to work...